### PR TITLE
feat: Rename lifecycle hook events

### PR DESCRIPTION
**Pull request by javieri-empathy**
EX-7580

This renames the agreed events to match the new shape.

### CODE REVIEW

**Comment by LuisMartinez15** [comment]
I assume you are only renaming the events that are attached to a vue's lifecycle hook. The `ExtraParamsInitialized` is only emitted when mounting the `ExtraParams` component and, as such, I think it should also be renamed to `ExtraParamsMountedHook`. On a side note, as the component doesn't render anything, can't we move the logic inside the `mounted` hook to the `created` one?

**Comment by javieri-empathy** [comment]
> I assume you are only renaming the events that are attached to a vue's lifecycle hook. The `ExtraParamsInitialized` is only emitted when mounting the `ExtraParams` component and, as such, I think it should also be renamed to `ExtraParamsMountedHook`. On a side note, as the component doesn't render anything, can't we move the logic inside the `mounted` hook to the `created` one?

I would't rename the `ExtraParamsInitialized`. This is one of the special cases that the team agreed on not renaming (proof below). Moreover, it doesn't mean that all events emitted from a lifecycle hook have to fall under the lifecycle hook category. For instance, DataProvided events.
image

I agree on moving the mounted -> created hook. I think it works properly. I don't see any duplicated requests, but take a careful second look please.

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
**praise:** thanks for splitting the task in multiple and smaller pull requests ❤️

File: packages/x-components/src/x-modules/queries-preview/wiring.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const queriesPreviewWiring = createWiring({
   QueryPreviewRequestChanged: {
     fetchAndSaveQueryPreviewWire
   },
-  QueryPreviewRemoved: {
```
[the remark this answers is not in the document]
Comment (javieri-empathy) [reply packages/x-components/src/x-modules/queries-preview/wiring.ts:59]: Yes, there is a task in the backlog for the next sprint about it.

**Review by LuisMartinez15 — APPROVED** [review]
**praise:** good job!

**Review by tiborux — APPROVED** [review]

**Pull request merged**